solr/CHANGES.txt

```diff
@@ -42,6 +42,9 @@ Bug Fixes
 * SOLR-17098: ZK Credentials and ACLs are no longer sent to all ZK Servers when using Streaming Expressions.
   They will only be used when sent to the default ZK Host. (Houston Putman, Jan Høydahl, David Smiley, Gus Heck, Qing Xu)
 
+* SOLR-17120: Fix NullPointerException in UpdateLog.applyOlderUpdates that can occur if there are multiple partial
+  updates of the same document in separate requests using commitWithin. (Calvin Smith, Christine Poerschke)
+
```

Comment:
Just noting that `main` and `branch_9x` and `branch_9_5` (if releasing after 8.11.3) don't have a 8.11.3 section as yet but would presumably get it in bulk as part of the 8.11.3 release process i.e.  no need for individual tickets to add the entries incrementally on those branches.